pom.xml

```diff
@@ -14,8 +14,8 @@
         <jenkins.version>2.112</jenkins.version>
         <java.level>8</java.level>
     </properties>
-    <name>JDK Tool Plugin</name>
-    <description>Allows the JDK tool to be installed via download from Oracle's website.</description>
+    <name>Oracle Java SE Development Kit Installer Plugin</name>
```

Comment:
too long? It will appear in the dependencies list on the plugin website, e.g. https://plugins.jenkins.io/ownership

Reply:
🤷‍♂  Seems fine. Maintainers need to update their core dependency to not pull in unnecessary plugins.